tests/conftest.py

```diff
@@ -851,3 +888,41 @@ def mock_events(api_url):
         status=200,
         body=response_body
     )
+
+
+@pytest.fixture
+def mock_upgrade(api_url, account_id, app_id):
+    account = {
+        "account_id": account_id,
+        "email_address": "ben.bitdiddle1861@gmail.com",
+        "id": account_id,
+        "name": "Ben Bitdiddle",
+        "object": "account",
+        "provider": "gmail",
+        "organization_unit": "label",
+        "billing_state": "trial",
```

Comment:
Can you change this to `cancelled` or `deleted` please? See https://github.com/nylas/nylas-python/pull/62/files#diff-484462fced51d1a06b1d93b4a44dd535R83